reader.go

```diff
@@ -301,6 +301,13 @@ func decodeLineOfMasterPlaylist(p *MasterPlaylist, state *decodingState, line st
 				state.variant.Captions = v
 			case "NAME":
 				state.variant.Name = v
+			case "FRAME-RATE":
+				var val float64
+				val, err := strconv.ParseFloat(v, 64)
```

Comment:
@cnguyenWurl - do we care about precision of FrameRate or its ok?

Reply:
@cnguyenWurl - I noticed we are using upto 3 decimals below when writing to the manifest, so ignore above comment.

Reply:
Just FYI. The spec said that we need 3 decimal precisions for frame-rate so I had to make sure it is properly defined.